src/App.js

```diff
@@ -14,12 +14,12 @@ import { CookieConsentProvider } from "./Components/CookieConsentContext";
 
 export const helmetContext = {};
 
-export function App() {
+export function App({ callback }) {
```

Comment:
I know that https://github.com/reactwg/react-18/discussions/5 suggest to name this prop `callback`. But I find it to be quite generic and from reading I don't really know what it suppose to do.

How about something like `onRenderCompletedCallback` or something similar?